SU2_CFD/src/variables/CNEMOEulerVariable.cpp

```diff
@@ -43,28 +43,28 @@ CNEMOEulerVariable::CNEMOEulerVariable(su2double val_pressure,
                                                                     ndim,
                                                                     nvar,
                                                                     config   ),
-                                      Gradient_Reconstruction(config->GetReconstructionGradientRequired() ? Gradient_Aux : Gradient) {
+                                      Gradient_Reconstruction(config->GetReconstructionGradientRequired() ? Gradient_Aux : Gradient_Primitive) {

```

Comment:
Need to check this...Grad Prim is not yet implemented for NEMO_merge branch.  Its on the way...